package.json

```
@@ -0,0 +1,29 @@
+{
+  "name": "optimizer",
+  "version": "0.0.1",
+  "description": "This library will optimize the AsyncAPI specification file.",
+  "main": "index.js",
```

Output will be in `./lib` so you should change it to (and also add types for TS):

```suggestion
  "main": "./lib/index.js",
  "types": "./lib/index.d.ts",
```